### ISSUE TITLE
[enhancement request] - Provide postgres branch

### ISSUE THREAD

**Issue by mtdeguzis** [jjethwa/icinga2#132]
Is there any chance of having a branch or a Dockerfile pointing towards a postgres variant of this?

**Comment by bebehei** [comment]
No, sorry. And there won't be such a thing in near future. Currently the MySQL server is running inside the container together with icinga2. It makes no sense installing postgres there, too. The benefit is almost zero, while shifting the complexity to a higher level.

In #112, I'm currently working on replacing the hardcoded MySQL server into an external resource (e.g.: another container). It may be the case, that postgres support might become bearable after merge.

**Comment by mtdeguzis** [comment]
Ok, i'll do my best to replace the mysql pieces with postgres ones.

**Comment by jjethwa** [comment]
Hi @mtdeguzis 

The way forward is what @bebehei suggested.  This image repo was created before there were container orchestration tools, so to make things easier for users, I set up MySQL to run inside the same container.  Postgres didn't have on par performance at that time as well, but it's caught up.   @bebehei  is working on a PR that all allow you to use an external MySQL instance.  Once it's merged, it should be easier to set things up for Postgres.   We'd love to have your help on that if you can 😃
